src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const updatedArray = sourceString
+    .split(';')
+    .map((item) => {
+      return item.split(':');
+    })
+    .filter((item) => item.length > 1)
+    .map((item) => {
+      return [item[0].trim(), item[1].trim()];
+    });
+
+  const finalObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `finalObject`. It will decrease code readability. Use more descriptive names that reflect the purpose of the variable, such as `styleObject` or `cssProperties`.